backend/canisters/user_index/api/src/queries/check_username.rs

```diff
@@ -1,12 +1,15 @@
 use candid::CandidType;
 use serde::{Deserialize, Serialize};
+use ts_gen::ts_export;
 
 #[derive(CandidType, Serialize, Deserialize, Debug)]
+#[ts_export(user_index, check_username)]
```

Comment:
Is it possible to do this with a macro which extracts user_index and check_username from context rather than having to state them explicitly?

Reply:
No. I spent a while digging into that. If a macro could detect the file it was in then it would be possible, but it can't